pkgs/development/python-modules/numpy/default.nix

```diff
@@ -54,6 +56,23 @@ in buildPythonPackage rec {
     # Patching of numpy.distutils is needed to prevent it from undoing the
     # patch to distutils.
     ./numpy-distutils-C++.patch
+    # f2py.f90mod_rules generates code with invalid function pointer conversions, which are
+    # clang 16 makes an error by default.
+    (fetchpatch {
+      url = "https://github.com/numpy/numpy/pull/24212.patch";
```

Comment:
I think we should use the commit link in case the PR has to be amended:

https://github.com/numpy/numpy/commit/609fee4324f3521d81a3454f5fcc33abb0d3761e.patch

Reply:
I switched it to use the commit link.